README.md

```diff
@@ -2,12 +2,58 @@
 
 Open-source 4G EnodeB and User emulators developed by [Software Radio Systems (SRS)](https://www.srslte.com/).
 
-## Usage
+## Installation and configuration
```

Comment:
I think the `usage` wording is pretty universal

Reply:
Reverted.